src/posts/xarray-biology/index.md

```diff
@@ -0,0 +1,135 @@
+
+## Call To Action
+
+`Xarray` has fantastic potential to accelerate our ability to perform and share biological analyses and accelerate discovery. To realize this potential, there are a few things we as a community need to do.
+
+The first thing is to consider what we mean by “community”. There are two separate communities with some overlap: `Xarray` developers and biologists interested in using `Xarray`.
+
+### What can Xarray Do
+
+As a community of contributors to `Xarray`, we need to:
+
+- **Support**: Be responsive to technical limitations and bugs in Xarray that affect biologists
+- **Educate**: cross-post about improvements in Xarray to spaces with biologists
+
+### What can Biologists Do?
+
+**Try** using `Xarray` for your data. Find the smallest, most straightforward way to get your data into Xarray and start using it.
+
+Make sure to **ask for help** when you need it. To do so:
+
+- Post on forums (e.g., [image.sc](https://forum.image.sc/), or the [Xarray Github Discussions](https://github.com/pydata/xarray/discussions))
+- Join the new Xarray for Biology office hours [TODO LINK]
+- Book one on one time with Ian [TODO LINK]
+
+Share how you did it with your colleagues and report bugs and feature requests to Xarray.
+
+If you have a success story, then you should share it! For example, via a short blog post here.
+
+### What can the Xarray-Bio Community do?
+
+Anyone already using `Xarray` to work with biology data is uniquely well-positioned to help speed adoption. We have three core tasks to grow the number of us.
+
+**Build** and then **share** tools that use `Xarray` to do useful things, such as building domain-specific visualization tools that use Xarray metadata to build the visualizations. Or solving any problems that you run into, if it bothers you, it likely bothers someone else and they’ll be happy to use your solution.
+
+**Contribute** by commenting on Xarray issues with how they affect biology use cases, and working on bug fixes and new features.
+
+**Support** other biologists learning to use Xarray. Respond to forum posts and help budding users, write and share small examples of using Xarray with biology data. Teach tutorials to your peers.
+
+My current role is an “Xarray community Developer” focusing on biological applications. So for my part, I’m always happy to talk to you about whether Xarray might be a good fit for your biology data. Please reach out if you have a question! I’m `@ianhi` on most platforms. You can also join our new Xarray in Biology office hours [LINK], or book some time with me to talk Xarray and Biology [here](https://calendly.com/ian-earthmover/30min).
```

Comment:
Your role is an important part of the narrative arc of the post.

Reply:
Agree with this. Let's move your own introduction to the intro. The arc could be:

- Hi, I'm Ian. I'm the new Xarray community dev, I'll be focusing on bio applications. btw, thanks CZI for the funding.
- In this post, I'll be surveying the landscape of Xarray for bio applications. This post is for you if: ...